.github/dependabot.yml

```diff
@@ -5,7 +5,11 @@ updates:
     schedule:
       interval: "daily"
     open-pull-requests-limit: 10
+    groups:
+      patches:
+        update-types:
+          - "patch"
```

Comment:
IMO we could also do this for minor?

Reply:
I wanted to start with this, as I tend to check release notes of minor releases more thoroughly. Sometime we change some of our code when they add a new feature.